mlocal/checks/project.chk

```diff
@@ -7,10 +7,15 @@ set -e
 makecond="$builddir/Makefile.cond"
 echo "" > $makecond
 
+########################
+# git version
+########################
+git_version=`git describe --match 'v[0-9]*' --long --dirty --always`
+
 config_add_def PACKAGE_NAME \"singularity\"
 config_add_def PACKAGE_TARNAME \"singularity\"
-config_add_def PACKAGE_VERSION \"3.0.0\"
-config_add_def PACKAGE_STRING \"singularity 3.0.0\"
+config_add_def PACKAGE_VERSION \"$git_version\"
+config_add_def PACKAGE_STRING \"singularity $git_version\"
 config_add_def PACKAGE_BUGREPORT \"support@sylabs.io\"
```

Comment:
Question ... Should it be this email address for the OSS version? Or should for OSS it be the mailing list?